### DISCUSSION TITLE
Can I start two async runtimes in two separate std thread?

### DISCUSSION THREAD

**Discussion by linrongbin16** [tokio-rs/tokio#6848] [Q&A] [answered]
Hi,

I want to create two runtimes in a command line, let's say A and B.

A is for working for IO (include keyboard/mouse events through crossterm EventStream) and some background jobs.

B is for V8 javascript engine, because V8 is designed to run in a single thread, the thread models seems not compatible with rust async. But some javascript language features such as `Promise` and `async` need to use rust async jobs, so js runtime like deno usually spawn them to tokio's tasks, that's why I also want to create B to run with V8 engine.

Is that possible?

**Comment by Darksonn** [comment, marked as answer]
Yes, you can create as many runtimes as you want via the `tokio::runtime::Builder` builder.

**Reply by linrongbin16** [reply]
Thanks for your reply!
